mu4/domain/notation/imasternotation.h

```diff
@@ -0,0 +1,47 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FIT-0NESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_DOMAIN_IMASTERNOTATION_H
+#define MU_DOMAIN_IMASTERNOTATION_H
+
+#include "modularity/imoduleexport.h"
+#include "inotation.h"
+#include "ret.h"
+#include "io/path.h"
+
+namespace mu {
+namespace domain {
+namespace notation {
+class IMasterNotation : virtual public INotation
+{
+public:
+    virtual Ret load(const io::path& path) = 0;
+    virtual io::path path() const = 0;
+
+    virtual Ret createNew(const ScoreCreateOptions& scoreInfo) = 0;
+
+    virtual std::vector<INotationPtr> parts() const = 0;
```

Comment:
Better named `excerpts`, not `parts`

Reply:
Can you explain to me, why "excerpts" is better naming for this method?

Reply:
Because we have `Part` object in the DOM (`class Part` in the libmscore). This method returned not this objects.

Reply:
renamed